source/Target/CMakeLists.txt

```diff
@@ -1,3 +1,19 @@
+include_directories(../Plugins/Process/Utility)
+
+set(REFLECTION_LIB "swiftReflection")
+
+if(LLVM_HOST_TRIPLE MATCHES "^((\\?!linux).)*$")
```

Comment:
I think it would be better to use MATCHES ".*-apple-.*" here, because Swift also builds on Windows and other platforms.

Reply:
ok! I'll change that now!

Reply:
changed to if(CMAKE_SYSTEM_NAME MATCHES "Darwin")